### PR TITLE
Remove monitoring dto and related attributes

### PR DESCRIPTION
**Pull request by IoannisLafiotis**
- Removed the Monitoring DTO and the rest of the objects related to that
- Removed the Monirtoring references 
- Removed the the tests that are going along with it
- Run the tests and compiled the code with no errors
- Removed the migration references for monitoring just to avoid conficts

### CODE REVIEW

File: src/main/resources/db/changelog/changeset/0001_initsql/initTables.sql [hunk 1]
```diff
@@ -83,23 +83,4 @@ INCREMENT BY 1
 MINVALUE = 0
 NOMAXVALUE
 START WITH 0
-CACHE 100;
-CREATE TABLE `userservice`.`session_monitoring` (
```
Comment (tkuzynow) [inline src/main/resources/db/changelog/changeset/0001_initsql/initTables.sql]: we need to have a migration to drop these tables, if they are not needed

File: src/test/java/de/caritas/cob/userservice/api/port/out/UserChatRepositoryTest.java [hunk 2]
```diff
@@ -29,39 +25,39 @@ class UserChatRepositoryTest {
...
-    // when
-    var findByChatAndUser = chatUserRepository.findByChatAndUser(chat, user).orElseThrow();
-
-    // then
-    assertThat(findByChatAndUser.getId()).isEqualTo(chatUser.getId());
-    assertThat(findByChatAndUser.getUser()).isEqualTo(chatUser.getUser());
-    assertThat(findByChatAndUser.getChat()).isEqualTo(chatUser.getChat());
-  }
-
+  /*
```
Comment (tkuzynow) [inline src/test/java/de/caritas/cob/userservice/api/port/out/UserChatRepositoryTest.java]: this should be not commented

File: src/main/java/de/caritas/cob/userservice/api/adapters/web/controller/UserController.java [hunk 3]
```diff
@@ -771,71 +766,6 @@ public ResponseEntity<Void> sendReassignmentNotification(
...
   }
 
-  /**
-   * Returns the monitoring for the given session.
-   *
-   * @param sessionId Session Id (required)
-   * @return {@link ResponseEntity} containing {@link MonitoringDTO}
-   */
-  @Override
-  public ResponseEntity<MonitoringDTO> getMonitoring(@PathVariable Long sessionId) {
```
Comment (tkuzynow) [inline src/main/java/de/caritas/cob/userservice/api/adapters/web/controller/UserController.java:781]: please make sure that on the frontend these endpoints are not used

File: api/userservice.yaml [hunk 4]
```diff
@@ -933,39 +933,6 @@ paths:
           description: INTERNAL SERVER ERROR - server encountered unexpected condition
       security:
         - Bearer: [ ]
-  /users/sessions/monitoring/{sessionId}:
```
Comment (tkuzynow) [inline api/userservice.yaml:936]: please confirm with Carlos that this is not used on the FE

File: api/userservice.yaml [hunk 5]
```diff
@@ -933,39 +933,6 @@ paths:
...
-    put:
-      tags:
```
Comment (tkuzynow) [inline api/userservice.yaml:938]: We need to install it on the cluster (all services containing these changes) and have smoke tested. Please ask @niklasb99  for support if needed.

File: api/userservice.yaml (hunk 4 above)
Comment (dodalovicgran) [reply api/userservice.yaml:936]: @CarlosSoares Anything you can help with?

Comment (CarlosSoares) [reply api/userservice.yaml:936]: This has already been removed by Rebecca in PR https://github.com/Onlineberatung/onlineBeratung-frontend/pull/891

**Review by tkuzynow — APPROVED** [review]

**Pull request merged**